renderer/components/load/import_keys.js

```diff
@@ -7,6 +7,7 @@ const ImportKeys = ({onSetKeysAndAddresses, onBack}) => {
     const privateKeyList = useRef()
     const handleClickNext = () => {
         const list = privateKeyList.current.value.split("\n")
+        list = [...new Set(list)]
```

Comment:
@AbdelSallam since `list` is being changed, should be a `let` now instead of a `const`.